.changeset/short-frogs-drop.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-graphql': patch
+---
+
+Upgrade `@graphql-codegen/cli` to latest.
```

Comment:
Actually no need to include a changeset since it's a dev dep. The bump would happen if we regenerate new code using the bumped version.

Reply:
I have removed the changeset. 👍